python/src/multi_agent_orchestrator/classifiers/bedrock_classifier.py

```diff
@@ -76,18 +76,24 @@ async def process_request(self,
             content=[{"text": input_text}]
         )
 
+        toolConfig = {
+            "tools": self.tools,
+        }
+
+        # ToolChoice is only supported by Anthropic Claude 3 models and by Mistral AI Mistral Large.
+        # https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_ToolChoice.html
+        if "anthropic" in self.model_id or 'mistral-large' in self.model_id:
+            toolConfig['toolChoice'] = {
+                "tool": {
+                    "name": "analyzePrompt",
```

Comment:
I would throw an error is another model is used with tools, otherwise one would not know that the tool is not used.

Reply:
The problem is that some models can use the tool but don't support toolChoice, which about forcing to use the tool.